code/modules/projectiles/guns/projectile/automatic.dm

```diff
@@ -88,6 +88,9 @@
 	burst_size = 2
 	pin = /obj/item/device/firing_pin/implant/pindicate
 
+/obj/item/weapon/gun/projectile/automatic/c20r/unrestricted
+	pin = /obj/item/device/firing_pin

```

Comment:
No @Miauw62 this is a definition.
You cant have objects in a definition.

Also 2 lines above ```pin = /obj/item/device/firing_pin/implant/pindicate```

Ignore Miauw, he's a silly.